packages/contracts/test/registry/plugin-repo-registry.ts

```diff
@@ -19,23 +23,36 @@ describe('Aragon-Plugin-Registry', function () {
   const PLUGIN_REGISTER_PERMISSION_ID = ethers.utils.id(
     'PLUGIN_REGISTER_PERMISSION'
   );
-  const pluginRepoName = 'my-pluginRepo';
+
+  const REGISTER_ENS_SUBDOMAIN_PERMISSION_ID = ethers.utils.id(
+    'REGISTER_ENS_SUBDOMAIN_PERMISSION'
+  );
+
+  const topLevelDomain = 'dao.eth';
```

Comment:
It is likely that we will use `dao-plugin.eth` (see https://discord.com/channels/672466989217873929/1031869502587158568/1031869505175040070).
You may want to adapt it for clarity.